app/src/main/java/com/conupods/Settings/SettingsActivity.java

```diff
@@ -0,0 +1,84 @@
+package com.conupods.Settings;
```

Comment:
This View should be in the Views folder

Reply:
The one inside OutdoorMaps? Although settings are also related to Indoor activities?

Reply:
Done for all activities